spec/support/features/webauthn_helper.rb

```diff
@@ -147,8 +147,9 @@ def credential_public_key
     HEREDOC
   end
 
+  # Deleting the trailing newline is necessary to prevent Capybara from auto-submitting forms
   def attestation_object
-    <<~HEREDOC
+    <<~HEREDOC.delete("\n")
```

Comment:
There's a perfect method for this, and it has the most adorable name, I strongly am in favor of:

```suggestion
    <<~HEREDOC.chomp
```